functions/streamtoidv/utils.py

```diff
@@ -0,0 +1,28 @@
+__version__ = '0.0.1'
+
+import os
+import logging
+import time
+from datetime import datetime
+from logging import NullHandler
+
+
+def set_stream_logger(name="cis-validator", level=logging.INFO, format_string=None):
+    """
+    :Stream logger class borrowed from https://github.com/threatresponse/aws_ir
```

Comment:
need to reflect that in license/attribution - https://github.com/ThreatResponse/aws_ir/blob/master/LICENSE.txt